### ISSUE TITLE
Simplify installation path for DW stack

### ISSUE THREAD

**Issue by anishasthana** [opendatahub-io/codeflare-operator#18]
There is currently no simple set of instructions that users can follow for installation of the Distributed Workloads stack using the v2 operator. We should remedy that.

**Comment by dimakis** [comment]
@KPostOffice you were working on this right?

**Comment by KPostOffice** [comment]
Yeah, I opened a PR in distributed workloads [here](https://github.com/opendatahub-io/distributed-workloads/pull/150)

**Comment by dimakis** [comment]
@anishasthana, I had already asked @KPostOffice to work on this. Thanks for opening the issue 👍
